### ISSUE TITLE
Seems to be a bug

### ISSUE THREAD

**Issue by poppik** [lammps/lammps#820]
There is a possible copy/paste bug in ComputeChunkAtom::atom2binsphere() function.
Current code goes as follows:
```
    xremap = x[i][0];
    if (periodicity[0]) {
      if (xremap < boxlo[0]) xremap += prd[0];
      if (xremap >= boxhi[0]) xremap -= prd[0];
    }
    yremap = x[i][1];
    if (periodicity[1]) {
      if (xremap < boxlo[1]) yremap += prd[1];
      if (xremap >= boxhi[1]) yremap -= prd[1];
    }
    zremap = x[i][2];
    if (periodicity[2]) {
      if (xremap < boxlo[2]) zremap += prd[2];
      if (xremap >= boxhi[2]) zremap -= prd[2];
    }
```
with **xremap** properly changed it should look like this:
```
    xremap = x[i][0];
    if (periodicity[0]) {
      if (xremap < boxlo[0]) xremap += prd[0];
      if (xremap >= boxhi[0]) xremap -= prd[0];
    }
    yremap = x[i][1];
    if (periodicity[1]) {
      if (yremap < boxlo[1]) yremap += prd[1];
      if (yremap >= boxhi[1]) yremap -= prd[1];
    }
    zremap = x[i][2];
    if (periodicity[2]) {
      if (zremap < boxlo[2]) zremap += prd[2];
      if (zremap >= boxhi[2]) zremap -= prd[2];
    }
```

**Comment by stanmoore1** [comment]
This does look like a bug. @poppik can you submit a PR?

**Comment by akohlmey** [comment]
no need. this is now included as commit b1d3b56a17ae4ccf1fd728209d5f4d66ed2cbfaa in PR #815
